website/docs/guides/markdown-features/markdown-features-code-blocks.mdx

```diff
@@ -142,6 +146,8 @@ html[data-theme='dark'] .docusaurus-highlight-code-line {
 }
 ```
 
+## Multiple line highlighting {#multiple-line-highlighting}
```

Comment:
I think the structure should be more like:

```md
## Line highlighting
### Single line highlighting
### Multiple line highlighting
### Highlighting with comments
```

Also, be ware that the section titles should be in sentence case.

Reply:
Good catch, thank you for noticing. It was fixed.